haproxy/Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM haproxy:2.3.19
+FROM haproxy:2.6
```

Comment:
I've chosen to not lock down the service pack version. This introduces a change of random build failures but means that we keep up to date with patches without any work. Let me know how you feel about this tradeoff.